### PR TITLE
update typings for leading trim, hanging punctuation, hanging list

### PR DESCRIPTION
**Pull request by joanna-chen**
https://app.asana.com/0/1203061614517911/1204148478726846/f

### CODE REVIEW

File: plugin-api.d.ts [hunk 1]
```diff
@@ -822,26 +829,14 @@ interface Easing {
     | 'EASE_OUT_BACK'
     | 'EASE_IN_AND_OUT_BACK'
     | 'CUSTOM_CUBIC_BEZIER'
-    | 'GENTLE'
-    | 'QUICK'
-    | 'BOUNCY'
-    | 'SLOW'
-    | 'CUSTOM_SPRING'
   readonly easingFunctionCubicBezier?: EasingFunctionBezier
-  readonly easingFunctionSpring?: EasingFunctionSpring
```
Comment (gh-owongfigma) [inline plugin-api.d.ts:831]: was this meant to be removed as part of this PR?

Comment (joanna-chen) [reply plugin-api.d.ts]: ah oops, need to rebase

File: plugin-api.d.ts [hunk 2]
```diff
@@ -454,8 +458,11 @@ declare type StyleChangeProperty =
   | 'textDecoration'
   | 'letterSpacing'
   | 'lineHeight'
+  | 'leadingTrim'
   | 'paragraphIndent'
   | 'paragraphSpacing'
```
Comment (gh-owongfigma) [inline plugin-api.d.ts:463]: listSpacing is a style property too I think (it should appear basically everywhere paragraph spacing does) (thank you!!)

**Review by gh-owongfigma — APPROVED** [review]
lgtm! just one thing about list spacing

**Pull request merged**